File: Intro-to-Python/Part1.py
password = input("Enter the password: ")
confirmation = input("Enter the password again: ")

print("Length 1: ",len(password))
print("Length 2: ",len(confirmation))

# if(len(password) == len(confirmation)):
#     print("Lengths match: True")
# else:
#     print("Lengths match: False")

print("Lengths match: ", len(password) == len(confirmation))


# if(password == confirmation):
#     print("Strings match: True")
# else:
#     print("Strings match: false")


print("Strings match: ", password == confirmation)

File: Intro-to-Java/Exercise-1/Part1.java
import java.util.Scanner;

public class Part1{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        String username = sc.next();
        String confirmation = sc.next();

        sc.close();

        System.out.println("Length 1: " + username.length());
        System.out.println("Length 2: " + confirmation.length());

        if(username.length() == confirmation.length()){
            System.out.println("Lengths match: true");
        }else{
            System.out.println("Lengths match: false");
        }

        if(username.equals(confirmation)){
            System.out.println("Strings match: true");
        }else{
            System.out.println("Strings match: false");
        }
        
    }
}

